src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -910,22 +892,61 @@ function GlobeControls(view, target, radius, options = {}) {
         };
     }());
 
+    this.configurationUi = {
```

Comment:
Couldn't you merge this `configurationUi` object and the `CONTROL_STATE` object (it may help readability)?